### DISCUSSION TITLE
make collapsible-sidebar-widget on issue always opened

### DISCUSSION THREAD

**Discussion by hogashi** [community/community#15612] [Projects and Issues]
Make the `collapsible-sidebar-widget` (e.g. linked Projects) always opened (or add default behavior configuration) !  I open this so many times to set issue's column status in Project Beta everyday.

![image](https://user-images.githubusercontent.com/12870472/165262066-53e301e4-1500-4254-8ab1-41e8e461bdc9.png)

**Comment by hianhianhian** [comment]
This would be a nice QoL to have!
